hw_diag/tasks.py

```diff
@@ -90,6 +90,10 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
         # No region found, put a dummy region in
         diagnostics['RE'] = "UN123"
 
+    # add frequency keys
+    diagnostics['FREQ'] = diagnostics['RE']  # friendly key
```

Comment:
didn't do a diagnostic for the time being as this already comes from Gateway_diagnostic in light_gateway branch. We will merge that soon.